agenta-backend/agenta_backend/services/llm_apps_service.py

```diff
@@ -55,13 +55,10 @@ def extract_result_from_response(response: dict):
                 value["data"] = str(value.get("data"))
 
             if "tree" in response:
-                trace_tree = (
-                    response["tree"][0]
-                    if isinstance(response.get("tree"), list)
-                    else {}
-                )
+                trace_tree = response.get("tree", {}).get("nodes", [])[0]
+
                 latency = (
-                    get_nested_value(trace_tree, ["time", "span"]) * 1_000_000
+                    get_nested_value(trace_tree, ["time", "span"]) / 1_000_000
```

Comment:
Do we still use time.span ? Is it still part of the schema ?

Reply:
Changes made.